src/software/new_geom/util/distance.cpp

```diff
@@ -11,3 +11,94 @@ double distance(const Point &first, const Line &second)
 {
     return distance(second, first);
 }
+
+double distance(const Point &first, const Point &second)
```

Comment:
We currently have ```Point::distanceFromOrigin()``` and ```Point::distanceFromPoint(const Point& p)``` in ```point.h``` - maybe we should remove those and replace their usages with this method?

Reply:
Sure, I'll do that in a new PR instead where I replace usages of `dist` with `distance` with the same issue #1136